src/main/resources/api.yaml

```diff
@@ -90,6 +90,117 @@ paths:
           description: User is not authorized
         404:
           description: Project not found
+  /projects/{projectId}/members:
+    get:
+      tags:
+        - projects
+      summary: Retrieves users belonging to the project
+      operationId: getUsersFromProject
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: projectId
+          description: Project ID
+          required: true
+          type: integer
+          format: int64
+      responses:
+        200:
+          description: Users retrieved successfully
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/ProjectMemberDto'
+        404:
+          description: Project not found
+    post:
+      tags:
+        - projects
+      summary: Adds a user to the project
+      operationId: addUserToProject
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: projectId
+          description: Project ID
+          required: true
+          type: integer
+          format: int64
+        - in: body
+          name: request
+          description: Information identifying the user
+          required: true
+          schema:
+            $ref: '#/definitions/UserIdentificationDto'
+      responses:
+        200:
+          description: User added successfully
+          schema:
+            $ref: '#/definitions/ProjectMemberDto'
+
+  /projects/{projectId}/members/{userId}/role:
+    put:
+      tags:
+        - projects
+      summary: Sets user's role in the project
+      operationId: setUserRole
+      consumes:
+        - application/json
+      parameters:
+        - in: path
+          name: projectId
+          description: Project ID
+          required: true
+          type: integer
+          format: int64
+        - in: path
+          name: userId
+          description: User ID
+          required: true
+          type: integer
+          format: int64
+        - in: body
```

Comment:
~Yaml generuje kod, który usuwa cały projekt~

O to to `request` się generuje jako string w Javie  i potem biedna Java dostaje `{ "request": "ADMIN" }` i jest smutna

Reply:
hm, bo w sumie enum nie może być dto (bo nie jest obiektem)
więc chyba trzeba by dodać coś w stylu:
```
schema:
  type: object
  properties: 
    role: 
      $ref: ///
``` 
(z dokładnością do syntaxu)
No i może usunąć dto z roleDto?

Reply:
Ale wtedy będziemy mieć paskudne importy w mapperze, bo będą dwa `UserRole` :(

Reply:
> hm, bo w sumie enum nie może być dto (bo nie jest obiektem)
więc chyba trzeba by dodać coś w stylu:
```
schema:
  type: object
  properties: 
    role: 
      $ref: ///
```
> (z dokładnością do syntaxu)

I wtedy do Javy przylatuje Object i trzeba coś czarować?

Reply:
Znaczy jest chyba generalnie plan na powrót do wizji z PUT